sources/HUBViewModelDiff.m

```diff
@@ -37,6 +38,12 @@
     return [indexPaths copy];
 }
 
+@interface  HUBViewModelDiff ()
+
+@property (nonatomic, assign) BOOL headerComponentIdentifierHasChanged;
```

Comment:
Couldn't the API be a tad confusing though if the model reports no deletions/inserts/deletions, but still claims to have changes?